pom.xml

```diff
@@ -221,7 +221,7 @@
                 <jdk>[17,)</jdk>
             </activation>
             <properties>
-                <!-- TODO: Remove this once google-java-fmt supports 17 or the plugin supports forking and run with these args built-in -->
+                <!-- TODO: Remove this once google-java-fmt supports 17 or the plugin supports forking -->
                 <argLine> --add-exports jdk.compiler/com.sun.tools.javac.api=ALL-UNNAMED --add-exports jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED --add-exports jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED --add-exports jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED --add-exports jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED</argLine>
```

Comment:
If the fix in google-java-format works, it should be possible to remove these flags right?